src/wallet/rpcwallet.cpp

```diff
@@ -249,8 +249,8 @@ static UniValue getnewminingaddress(const JSONRPCRequest& request)
 
     CKeyID dest = newKey.GetID();
     pwallet->LearnRelatedScripts(newKey, OutputType::LEGACY);
-
-    pwallet->SetAddressBook(dest, label, "receive");
+    
+    pwallet->SetAddressBook(dest, label, "basecoin");
```

Comment:
`setnonautomintaddress` uses the label `basecoin` for the address that is set to not be eligible for automint. Need to use a different label here.

Reply:
Hmm, the GUI has been creating miner addresses and formatting them on the correct format using the basecoin tag. Check `addressreceive.cpp` file, generateNewAddress method.

Reply:
Probably the reason why automint isn't working for many people.

Reply:
We need to be sync on this kind of points, the basecoin tag purpose was added 16 days ago on the GUI.

Reply:
Let just remove the mining addresses from the gui entirely. It was originally created by blondfrogs for the pools, but seems the pools chose to just continue with basecoin until mining to SA's is ready

Reply:
@furszy I know, I know.. I asked u to add that button.. :(

Reply:
🤷‍♂️ .

If mining pools continued with basecoin addresses and the goal is move to stealth addresses, why we are having "mining addresses" at all?

Not only GUI, should be removed from the RPC commands too.

Reply:
Would this discussion above invalidate #279 ?

Reply:
Yes @StrontiumZ38

Reply:
I believe the pools themselves are using the mining addresses based on seeing where the pool transactions are being distributed from, so for now,r emoving it from the GUI would be fine (and yes, that is my fault) We should check with @blondfrogs for more detailed info on the need of the Mining address.

Reply:
all good, we were all under big pressure on that time.

My point goes in the direction of: 
There is no need to keep a functionality and support it when it's not part of the main goal.
Only reason that comes to my mind is that mining pools don't have bech32 support.

But well, @blondfrogs we are all ears.

Reply:
getminingaddress was created, because I wanted to build a simple mechanism for pools to use to generate address that they could create coinbase transactions with. Pools run a check to make sure the the addresses are base58 encoded. This means the the normal getnewbasecoinaddress wasn't working, because bech32 address don't pass this check.

As far as I know, pools should be the only ones that are using this rpc call. everyone else should be able to use getnewbasecoinaddress

Reply:
I think removing it from the GUI completely is a great move. However, we should keep the rpc call until mining to Stealth Addresses is complete and implemented on the pool side.